guide/easy_install.md

```diff
@@ -0,0 +1,156 @@
+
+The script may be given the "-f" option. If you do, you are telling the script
+that you do not want any existing installation of ffmpeg to be overwritten with
+a newer one. This could be important to you. If you do use it, and the version
+you have installed is not compatible with moonfire, you will be told through
+a message. If you have no ffmpeg installed, the option is effectively ignored
+and the necessary version of ffmpeg will be installed.
+
+The setup script should only need to be run once (after `prep.config` has been
+created), although if you do a re-install of moonfire, in particular a much
+newer version, it is a good idea to run it again as requirements and pre-requisites
+may have changed. Running the script multiple times should not have any negative effects.
+
+*WARNING* It is quite possible that during the running of the setup script,
+in particular during the building of libavutil you will see several compiler
+warnings. This, while undesirable, is a direct result of the original
+developers not cleaning up the cause(s) of these warnings. They are, however,
+just warnings and will not affect correct functioning of moonfire.
+
+Once the setup is complete, two steps remain: building and then installing.
+There is a script for each of these scenarios, but since generally you would
+want to install after a succesul build, the build script automatically invokes
+the install script, unless specifically told not to.
+
+## Building
+
+The build script is involved like this:
+
+    $ scripts/build.sh
+
+This script will perform all steps necessary to build a complete moonfire
+setup. If there are no build errors, this script will then automatically
+invoke the install script (see below).
+
+There are two options you may pass to this script. The first is "-B" which
+means "build only". In other words, this will stop the automatic invocation
+of the instal script. The other option available is "-t" and it causes the
+script to ignore the results of any tests. In other words, even if tests
+fail, the build phase will be considered succesful. This can occasionally
+be useful if you are doing development, and have temporarily broken one
+or more test, but want to proceed anyway.
+
+## Installing
+
+The install step is performed by the script that can be manually invoked
+like this:
+
+    $ scripts/install.sh
+
+This script will copy various files resulting from the build to the correct
+locations. It will also create a "service configuration" for systemctl that
+can be used to control moonfire. This service confuration can be prevented
+by using the "-s" option to this script. It will also prevent the automatic
+start of this configuration.
+
+
+## Configuration variables
+
+Although not all listed in the default prep.config file, these are the
+available configuration variable and their defaults. In the most frequent
+scenarios you will probably only change the port to 80 (assuming you
```

Comment:
Does changing this to 80 work? Ports < 1024 are privileged, and I don't see anything here that would allow the moonfire-nvr user to bind to that port.

Eventually I hope to support passing the already-bound socket from systemd, which I gather is what the cool kids do. For now, I just use port 8080 in my own setup.